blocks/math/include/gnuradio-4.0/math/Math.hpp

```diff
@@ -0,0 +1,67 @@
+#ifndef GNURADIO_MATH_HPP
+#define GNURADIO_MATH_HPP
+
+#include <gnuradio-4.0/Block.hpp>
+#include <gnuradio-4.0/BlockRegistry.hpp>
+#include <gnuradio-4.0/DataSet.hpp>
+#include <gnuradio-4.0/meta/UncertainValue.hpp>
+
+namespace gr::blocks::math {
+
+namespace detail {
+template<typename T>
+T
+defaultValue() noexcept {
+    if constexpr (gr::arithmetic_or_complex_like<T> || gr::UncertainValueLike<T>) {
+        return static_cast<T>(1);
+    } else if constexpr (std::is_arithmetic_v<T>) {
+        return static_cast<T>(1);
+    } else {
+        return T{};
+    }
+}
+} // namespace detail
+
+template<typename T, char op>
+struct MathOpImpl : public gr::Block<MathOpImpl<T, op>, gr::PortInNamed<T, "in">, gr::PortOutNamed<T, "out">> {
```

Comment:
Probably 

```cpp
PortIn<T> in;
PortOut<T> out;
```
would be more intuitive?

Reply:
Was part of the original block implementation and merge-API. Don't want to tackle the merge API right now. Maybe a non-issue but spent already quite some bit on the compile-time improvements and want to get this through for the benefit of the whole team.